src/components/group-management-menu/index.tsx

```diff
@@ -81,7 +81,8 @@ export class GroupManagementMenu extends React.Component<Properties, State> {
         side='bottom'
         alignMenu='end'
         onOpenChange={this.handleOpenChange}
-        trigger={<IconButton Icon={IconDotsHorizontal} size={32} isFilled onClick={() => {}} />}
+        className={'group-management-trigger'}
+        trigger={<IconDotsHorizontal size={24} isFilled />}
```

Comment:
Hmm... is this going to cause us problems if we ever change the icon button styles?

Reply:
Though, I can't really think of a fix. lol